README.md

```diff
@@ -17,7 +17,7 @@ This is my research and reflection journal for DGL 104 winter 2023
 - up to date with the new technologies.
 - want to develop one site for my village and local area about things in the places.
 
```

Comment:
You've got some good ideas in here, but we need to refocus on coding and make your goals more specific and measurable.

You say you want to create some new things with a learned technology, and to keep up to date with new technologies. I think these two together could be a good place to focus. What programming language are you thinking about using? I think we could use these two ideas, in combination with your programming language to come up with an idea that would meet these two ideas (making something new, and keeping up with new technologies) that could also serve to support a community.